modules/extensions-mule-support/src/main/java/org/mule/runtime/module/extension/mule/internal/loader/parser/MuleSdkOperationModelParser.java

```diff
@@ -213,7 +217,11 @@ public boolean isRouter() {
 
   @Override
   public boolean isConnected() {
-    // TODO: MULE-20077
+    return isConnected.getValue();
+  }
+
+  @Override
+  public boolean requiresOwnConnection() {
```

Comment:
this method should no longer be needed since everything now depends on the flavor/reconnection strategy

Reply:
Removed